### PR TITLE
Responsive Homepage & Header

### PR DESCRIPTION
**Pull request by dfosco**
- [x] Homepage
  + [x] Policies
  + [x] Footer
  + [x] Newsletter
  + [ ] ~~Shelf~~ (TODO in a separate PR)

- [x] Header 
   + [ ] ~~Desktop category display~~ (TODO in a separate PR)
   + [x] Fix event on `HeaderMenuButton` [#117](https://github.com/vtex-apps/theme/issues/117)
   + [x] Fix layout for `HeaderMenuButton`
   + [x] Responsive search bar

### CODE REVIEW

**Comment by dfosco** [comment]
@tamorim @brenoc Could you review this? We need to merge is as soon as possible so @afonso-praca doesn't have a lot of trouble to get the changes.

File: src/components/Header/index.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import ReactCSSTransitionGroup from 'react-addons-css-transition-group';
+import { Area } from 'sdk';
```
Comment (brenoc) [inline src/components/Header/index.js]: You need to update this.

You can do the exact thing that I'm doing here:
https://github.com/vtex-apps/theme/pull/120/files

Comment (dfosco) [inline src/components/Header/index.js]: @brenoc Should I change only the imports or the rest of the code changed as well?

```
+import { Area } from 'sdk';
```

Comment (brenoc) [inline src/components/Header/index.js]: Line 2 and 13 and 14 from the link I posted.

**Comment by dfosco** [comment]
@brenoc Done!

**Comment by dfosco** [comment]
@brenoc Done, but not really... changes are breaking the toolbelt

**Comment by dfosco** [comment]
@brenoc Weird, seeing that I copied directly your code: 

image

File: src/components/Header/index.js (hunk 1 above)
Comment (brenoc) [inline src/components/Header/index.js]: Hey:

```diff
-import { Area } from 'sdk';
+import { stores } from 'sdk';
```

**Comment by brenoc** [comment]
@dfosco See comment above!

**Comment by dfosco** [comment]
:ok_hand: :neutral_face:

**Comment by dfosco** [comment]
@brenoc Cool, now working

**Comment by brenoc** [comment]
@dfosco please fixup those last two commits

**Pull request merged**